frontend/public/components/routes.tsx

```diff
@@ -102,15 +100,31 @@ export const RouteStatus: React.SFC<RouteHostnameProps> = ({obj: route}) => {
     }
   });
 
-  return atLeastOneAdmitted
-    ? <span>
-      <div className="fa fa-check text-success co-m-route-status-icon" aria-hidden="true"></div>
+  return atLeastOneAdmitted ? 'Accepted' : 'Rejected';
+};
+
+export const RouteStatus: React.SFC<RouteHostnameProps> = ({obj: route}) => {
+  const status: string = routeStatus(route);
+
+  switch (status) {
+    case 'Pending':
+      return <span>
+        <div className="fa fa-hourglass-half co-m-route-status-icon" aria-hidden="true"></div>
+        Pending
+      </span>;
+    case 'Accepted':
+      return <span>
+        <div className="fa fa-check text-success co-m-route-status-icon" aria-hidden="true"></div>
       Accepted
-    </span>
-    : <span>
-      <div className="fa fa-times text-danger co-m-route-status-icon" aria-hidden="true"></div>
+      </span>;
+    case 'Rejected':
+      return <span>
+        <div className="fa fa-times text-danger co-m-route-status-icon" aria-hidden="true"></div>
       Rejected
-    </span>;
+      </span>;
+    default:
+      break;
```

Comment:
I'd remove the empty default case

Reply:
I get an ESLint error saying it expects a default for the switch

Reply:
OK I see we do this in other places as well. Disregard :)